packages/marshal/src/rankOrder.js

```diff
@@ -272,6 +275,7 @@ export const makeComparatorKit = (compareRemotables = (_x, _y) => 0) => {
         throw Fail`Unrecognized passStyle: ${q(leftStyle)}`;
       }
     }
+    /* eslint-enable */
```

Comment:
does this re-enable everything that's been disabled? consider being more explicit

Reply:
It does ("[_`/* eslint-enable */` without any specific rules listed causes all disabled rules to be re-enabled._](https://eslint.org/docs/latest/use/configure/rules#using-configuration-comments-1)"), and being more explicit is bad here because it can enable rules that were never on.